modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/impl/TestMvPartitionStorage.java

```diff
@@ -55,6 +56,9 @@ public class TestMvPartitionStorage implements MvPartitionStorage {
     /** Preserved {@link LocalLocker} instance to allow nested calls of {@link #runConsistently(WriteClosure)}. */
     private static final ThreadLocal<LocalLocker> THREAD_LOCAL_LOCKER = new ThreadLocal<>();
 
+    private static final AtomicLongFieldUpdater<TestMvPartitionStorage> ESTIMATED_SIZE_UPDATER =
```

Comment:
Why even bother, it's a test storage

Reply:
It actually looks better, I think, because in some places you can still handle this as a `long`, without using `AtomicLong#get`